apps/server/src/modules/tool/controller/dto/request/context.params.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum } from 'class-validator';
+import { ToolContextType } from '../../../interface';
+
+export class ContextParams {
```

Comment:
i think this name will will clash with my pr. Can you rename it to something like ContextTypeParams?